.github/workflows/build.yml

```diff
@@ -7,10 +7,10 @@ on:
 name: Build
 
 jobs:
-  host_tests:
+  macos_and_windows:
     strategy:
       matrix:
-        os: [macos-latest, windows-2019]
+        os: [macos-latest, windows-2019, macos-13]
```

Comment:
Why can't we build a universal build of zstd? 

It seems officially supported: https://github.com/facebook/zstd#support-for-fat-universal2-output

Reply:
We probably could, but I don't know how to build zstd without ninja as a universal build (and fwiw - that link you sent builds the zstd compression cli, not the shared library).  Building explicitly on x64 is simpler, easier to understand and requires less tooling (it just works out of the box).